.github/workflows/benchmark_pr.yml

```diff
@@ -0,0 +1,36 @@
+on:
+  pull_request:
+    types: [opened, reopened, edited, synchronize]
+
+jobs:
+  benchmark_pr_branch:
+    name: Continuous Benchmarking PRs with Bencher
+    if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository
```

Comment:
This means only those who can push to/open mrs on this repo (not mrs from another) will run the benchmarks, right? just worried about potential spam abuse

Reply:
Correct, forks won't trigger the workflow